Documentation/Configuration/Typo3ConfVars/DB.rst

```diff
@@ -138,6 +138,33 @@ the connection to the database:
         Name of the database/schema to connect to. Can be used with
         MySQL/MariaDB and PostgreSQL.
 
+    ..  _typo3ConfVars_db_connections_defaulttableoptions:
```

Comment:
```suggestion
```
It is not needed to introduce new anchors for confvals as the confval can already be referenced directly and the anchor would have no title. We only kept the old anchors for backward compability in case they have already been referenced